apps/anomalycor/.eslintrc.json

```diff
@@ -28,22 +28,11 @@
     "space-before-function-paren": "off",
     // for Meteor API's that rely on `this` context, e.g. Template.onCreated and publications
     "func-names": "off",
-    "prefer-arrow-callback": "off",
+    "prefer-arrow-callback": "off"
 
     // Vx Team modifications - Warn on rules that would require refactoring to implement.
     // We want to be able to turn these back into "error"'s at some point. However, for
     // our first pass, we'll only consider the checks that ESLint can auto-fix as errors.
     // https://eslint.org/docs/latest/use/configure/rules#rule-severities
```

Comment:
I left it there in case we did want to add future exceptions, but I can take it back out.

Reply:
I'd be a fan of taking it out. In my experience orphaned comments tend to go stale pretty quickly (e.g. the third sentence of that comment is no longer relevant) and just make it more difficult to understand what's going on in the future.

If we did want to leave it, we'll want to update the comment to point out that it's a placeholder for future exceptions, and that everything should be done to avoid adding any new ones. Something like:

```
    // Placeholder for Vx Team rule modifications
    // Add rules that we want to switch from errors to warnings here. Note that this
    // should be the last solution considered. There's almost always a better way for
    // the code to be written so that it satisfies the app's eslint ruleset. You can consult
    // the airbnb guidelines for hints on how to restructure your code to address
    // particular error messages here: https://airbnb.io/javascript/. If any exceptions
    // are added, the goal should be to switch them back into "error"'s as soon as
    // possible.
    // https://eslint.org/docs/latest/use/configure/rules#rule-severities
```